### PR TITLE
Lighthouse improvements [#575]

### PR DESCRIPTION
**Pull request by jordinl83**
Some improvements for PageSpeed:

* Load `main.js` in head with `deferr` tag
* Add `Cache-Control` header to js and css, given these are fingerprinted.
* Load roboto font from CDN.

### CODE REVIEW

**Comment by jordinl83** [comment]
LGTM

File: webpack.config.prod.js [hunk 1]
```diff
@@ -87,7 +87,8 @@ export default {
       template: path.resolve(__dirname, './src/index.ejs'),
       favicon: path.resolve(projectPath, './images/favicon.png'),
       minify: false,
-      inject: true,
+      inject: 'head',
+      scriptLoading: 'deferr',
```
Comment (dankohn) [inline webpack.config.prod.js:91]: It's `defer`.

**Comment by dankohn** [comment]
Can you say what the improvement is for the font? It's not obvious to me that the DNS lookup plus download of the font is faster than getting it from the Netlify servers. But it might be if it enables caching.

**Comment by jordinl83** [comment]
@dankohn The fonts from the CDN are not correctly being picked up, that explains the higher score... Also, the actual font used (Helvetica) does not look that different from Roboto, so I didn't notice the first time.

**Comment by dankohn** [comment]
Can you please write the score here before and after the pull request to maintain for posterity? Also, should we consider not getting fonts at all and just using the system font?

**Comment by jordinl83** [comment]
PageSpeed score varies quite a lot across different runs, so I don't think we can get very meaningful results.

I think loading `main.js` earlier with `defer` and not relying on third-party fonts are improvements though.

LGTM

**Pull request merged**